src/Countries/US/Maryland/MarylandIncome/HasMarylandIncome.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Maryland\MarylandIncome;
+
+use Illuminate\Database\Eloquent\Collection;
+
+trait HasMarylandIncome
+{
+    public function getAdjustedEarnings()
```

Comment:
i think this can be private since compute is overridden

Reply:
i thought so as well, but it has to be public for `BaseTax`.